apps/cookbook/src/app/examples/link-example/link-example.component.html

```diff
@@ -3,3 +3,6 @@ <h2>Sizes</h2>
 
 <h2>New tab</h2>
 <cookbook-link-example-new-tab #newTab></cookbook-link-example-new-tab>
+
+<h2>Button</h2>
+<cookbook-link-example-button #newTab></cookbook-link-example-button>
```

Comment:
Unsure why this is added to the other examples, but can most likely be omitted.
```suggestion
<cookbook-link-example-button></cookbook-link-example-button>
```